src/components/Modals/ConfirmTalkingPointModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { H4, P, Button, Modal, ModalBody, ModalFooter, ModalHeader, ModalCloseButton, LinkButton } from "@c1ds/components"
+import { Flex, Box } from "@chakra-ui/core"
+
+interface ConfirmTalkingPointModalProps {
+	isOpen: boolean
+	onClose: React.ComponentProps<typeof Modal>["onClose"]
+	onConfirm: () => void
+}
+
+export const ConfirmTalkingPointModal: React.FC<ConfirmTalkingPointModalProps> = (p: ConfirmTalkingPointModalProps) => (
+	<Modal isOpen={p.isOpen} onClose={p.onClose} isCentered={true} size="sm">
+		<ModalHeader>
+			<H4>Empty Talking Point</H4>
+		</ModalHeader>
+		<ModalCloseButton />
+		<ModalBody>
+			<P>The event is saving with the pre-determined Talking Points, Would you like to continue?</P>
```

Comment:
We probably need to confirm this verbiage as well. Our tester will probably mention something as well, but below is a small suggestion:
```suggestion
			<P>The event is saving with the pre-determined Talking Points. Would you like to continue?</P>
```

Reply:
I have confirmed the verbiage with Katherine before I create the PR but still thank you for the grammatical fix.